geometry_msgs/Pose pose
float32 cov_major
float32 cov_minor
